### PR TITLE
DOC: Add examples for filling missing data

### PR DESCRIPTION
**Pull request by chrystalchan-academic**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/user_guide/missing_data.rst [hunk 1]
```diff
@@ -239,6 +239,16 @@ of ways, which we illustrate:
    df2.fillna(0)
    df2["one"].fillna("missing")
 
+**Replace NA for columns of specific data types**
+
+.. ipython:: python
+
+nonDateDf = df.select_dtypes(exclude=['datetime'])
```
Comment (jreback) [inline doc/source/user_guide/missing_data.rst:246]: need to be indented. don't use camelCase, pref `non_date_df`

Comment (jreback) [reply doc/source/user_guide/missing_data.rst:246]: its not clear what you are trying to do here, can you add a comment & show the resulting frame (and the before one if needed); you are assigning in place so it won't show currently.

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by MarcoGorelli** [comment]
Hi @chrystalchan-academic - is this still active?

**Comment by MarcoGorelli** [comment]
closing for now, let us know if you'd like to continue working on this and we'll reopen

**Pull request closed without merging**